app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/confirmation_of_submission.scala.html

```diff
@@ -50,6 +50,29 @@
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 @isATrust = @{ justSubmittedReturn.subscribedDetails.isATrust }
 
+@completionDate =  @{justSubmittedReturn.completeReturn.fold(
+                        _.triageAnswers.fold(
```

Comment:
you don't need to fold here and below - `_.triageAnswers" shold be complete so you can get a non-optional completion date from all the cases

Reply:
ripped out the fold and did it via case analysis - much easier to read.